xLights/sequencer/Element.cpp

```diff
@@ -595,6 +595,18 @@ bool ModelElement::HasEffects() const
     return false;
 }
 
+int ModelElement::GetModelEffectCount() const
+{
+    int sum = std::accumulate(
+        mEffectLayers.begin(),
+        mEffectLayers.end(), 0,
+        [](int i, EffectLayer* l) {
```

Comment:
Is this a lambda function?  Wish I understood this new stuff.   I saw someone use a lambda at work and I looked it up and still don't understand it...lol.

Reply:
Yep just an anonymous function written right there - not that hard to understand.  std::accumulate takes a function among other things... the begin/end of the range to accumulate, etc.

That syntax looks weird at first but it is also not complicated.
[captures - none here - but it would be stuff the function needs that is not present in the parameters, but rather in the calling context](parameters/arguments){code body that does stuff on the parameters}